sts-mock/template.yaml

```diff
@@ -145,6 +145,11 @@ Resources:
             Method: post
             RestApiId: !Ref StsMockApi
       Role: !GetAtt TokenFunctionLambdaRole.Arn
+      Environment:
+        Variables:
+          # MOCK_STS_BASE_URL to be updated once available
```

Comment:
I'd suggest also passing through the AUD value here - it'll need to be dynamic per environment. 

The value to answer your question elsewhere will be:

https://mob-async-backend.review-b-async.<ENVIRONMENT>.account.gov.uk

Reply:
Ahhh ok, I didn't realise it will change per environment, but I should have 🤦 - thanks!

Reply:
Done - `ASYNC_BACKEND_BASE_URL`